src/bepasty/utils/upload.py

```diff
@@ -66,6 +66,7 @@ def filter_type(cls, ct, ct_hint, filename=None):
         Only allow some basic characters and shorten to 50 characters.
         """
         if not ct and filename:
+            mimetypes.add_type('application/x-asciinema-recording', '.cast')
```

Comment:
this doesn't look like the right place for this.

i suspect filter_type is a frequently called function and one does not need to call add_type more than once per process.

Reply:
Yes, I can do that in cli